tests/unit/conftest.py

```diff
@@ -241,7 +251,9 @@ def default_config(tmp_db_path, tmp_spam_domains_file):
             },
         },
         SEARCH_INDEX_PREFIX='zenodo-test-',
-        ZENODO_SPAM_DOMAINS_FILEPATH=tmp_spam_domains_file,
+        ZENODO_SPAM_DOMAINS_FORBIDDEN_LIST_FILEPATH
+        =tmp_spam_domains_forbidden_list_file,
```

Comment:
Hm, I think this one will probably get a complaint from pep8

Reply:
Got this autoformatted from my IDE, just tried with Black online and it seems a one-liner seems to work indeed